kratos/tests/cpp_tests/utilities/test_math_utils.cpp

```diff
@@ -870,5 +870,30 @@ namespace Kratos
             KRATOS_CHECK_EQUAL(a(1,1), 1.0);
         }
 
+        /** Checks if it calculates the Factorial
+         */
+        KRATOS_TEST_CASE_IN_SUITE(MathUtilsFactorial, KratosCoreFastSuite)
+        {
+            KRATOS_CHECK_EQUAL(MathUtils<double>::Factorial(0), 1);
+            KRATOS_CHECK_EQUAL(MathUtils<double>::Factorial(1), 1);
+            KRATOS_CHECK_EQUAL(MathUtils<double>::Factorial(2), 2);
+            KRATOS_CHECK_EQUAL(MathUtils<double>::Factorial(6), 720);
+            KRATOS_CHECK_EQUAL(MathUtils<double>::Factorial(8), 40320);
+        }
+
+        /** Checks if the exponential of a matrix is performed correctly
+         */
+        KRATOS_TEST_CASE_IN_SUITE(MathUtilsExponentialOfMatrix, KratosCoreFastSuite)
+        {
+            Matrix A(3, 3), exp_A(3, 3);
+            noalias(A) = IdentityMatrix(3);
+            A(1, 1) = 2.0;
+            A(2, 2) = 3.0;
+            MathUtils<double>::CalculateExponentialOfMatrix(A, exp_A);
+            KRATOS_CHECK_NEAR(exp_A(0, 0), std::exp(A(0, 0)), 1.0e-4);
+            KRATOS_CHECK_NEAR(exp_A(1, 1), std::exp(A(1, 1)), 1.0e-4);
+            KRATOS_CHECK_NEAR(exp_A(2, 2), std::exp(A(2, 2)), 1.0e-4);
```

Comment:
can you please use the matrix check macro?
And why such low tolerance?

Reply:
done it, BTW, How do you run the cpp tests of the CORE????

Reply:
`python kratos/python_scripts/run_cpp_tests.py`